.ci/atime/tests.R

```diff
@@ -119,6 +119,25 @@ test.list <- atime::atime_test_list(
       data.table:::setDT(L)
     },
     Slow = "c4a2085e35689a108d67dacb2f8261e4964d7e12", # Parent of the first commit in the PR that fixes the issue (https://github.com/Rdatatable/data.table/commit/7cc4da4c1c8e568f655ab5167922dcdb75953801)
-    Fast = "1872f473b20fdcddc5c1b35d79fe9229cd9a1d15") # Last commit in the PR that fixes the issue (https://github.com/Rdatatable/data.table/pull/5427/commits)
+    Fast = "1872f473b20fdcddc5c1b35d79fe9229cd9a1d15"), # Last commit in the PR that fixes the issue (https://github.com/Rdatatable/data.table/pull/5427/commits)
+
+  # Issue reported in: https://github.com/Rdatatable/data.table/issues/4200
+  # To be fixed in: https://github.com/Rdatatable/data.table/pull/4558
+  "DT[by] fixed in #4558" = atime::atime_test(
+    N = 10^seq(1, 20),
+    setup = {
+      d <- data.table(
+        id3 = sample( c(seq.int(N*0.9), sample( N*0.9, N*0.1, TRUE))),
+        v1 = sample(5L, N, TRUE),
+        v2 = sample(5L, N, TRUE)
+      )
+    },
+    expr = {
+      expr=data.table:::`[.data.table`(d, , (max(v1)-min(v2)), by = id3)
+    },
+    Before="905e4d302be0dc326684d7228ece88064321cc00", # Parent of the regression commit (https://github.com/Rdatatable/data.table/commit/c152ced0e5799acee1589910c69c1a2c6586b95d)
```

Comment:
Spaces before and after the `=` (lines 136, 138, 139, 140) would be nice (and perhaps for operators like `-` and `*` above too you can have spacing before and after)

Reply:
I agree. You should look at the nearby code and try to use the same style.

Reply:
also parenthesis around max-min are not necessary, please remove